src/XMakeBuildEngine/Definition/Project.cs

```diff
@@ -1191,6 +1186,11 @@ public List<ProvenanceResult> GetItemProvenance(string itemToMatch, string itemT
         /// </param>
         public List<ProvenanceResult> GetItemProvenance(ProjectItem item)
         {
+            if (item == null)
+            {
+                return Enumerable.Empty<ProvenanceResult>().ToList();
```

Comment:
Nope. I should have deleted that one. I was exploring how I can get a singleton empty collection without instantiating a new object. Exploration was futile :)